lua/entities/gmod_wire_pod.lua

```diff
@@ -369,7 +371,7 @@ function ENT:RCEject()
 end
 
 function ENT:TriggerInput( name, value )
-	if (name == "Lock") then
+		if (name == "Lock") then
```

Comment:
What happened here?

Reply:
I was experimenting to see if replacing the elseifs with a lookup table would be faster but it wasn't, i didnt catch that indentation mistake.